### PR TITLE
tools/radvd: Add script to run radvd on a tun interface

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

This adds a small wrapper script to configure and start radvd on a
tun interface.
This allows the use of router advertisements and global adresses with
the `native` target.

### Testing procedure

 - first create the tap interface as usual
   ```
    sudo dist/tools/tapsetup/tapsetup
   ```
 - now run radvd on the new tapbr0 interface
   ```
    sudo dist/tools/radvd/radvd.sh -c tapbr0 2001:db8::/64
   ```
 - Now run the `gnrc_networking` example on native:
   ```
    make -C examples/gnrc_networking all term
   ```
You should now see that the `native` node has received a global address
in `ifconfig`.
You should be able to reach this address from your host computer.

It may take very long for the native node to obtain the address.
If you are observing this, try turning off router advertisements of the
native node on the upstream interface by running

    ifconfig 6 -rtr_adv

Alternatively change `USEMODULE += gnrc_ipv6_router_default` to
`USEMODULE += gnrc_ipv6_default` in the project's `Makefile`.

### CODE REVIEW

**Comment by miri64** [comment]
IMHO this could benefit from the tooling bootstrapper proposed in #13609.

**Comment by gschorcht** [comment]
Works as expected.

File: dist/tools/radvd/radvd.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    if [ -n "${PID}" ]; then
+        kill ${PID}
+        rm ${PIDFILE}
+    fi
+}
+
+start_radvd() {
+    export TAP
+    export PREFIX
+    cat ${CURRENT_DIR}/radvd.conf | envsubst | radvd -C /dev/stdin -u ${SUDO_USER} -p ${PIDFILE}
```
Comment (miri64) [inline dist/tools/radvd/radvd.sh]: Not sure this is just a style thing, but
```suggestion
    radvd -C <(cat ${CURRENT_DIR}/radvd.conf | envsubst) -u ${SUDO_USER} -p ${PIDFILE}
```

seems cleaner to me.

Comment (benpicco) [reply dist/tools/radvd/radvd.sh]: `/bin/sh` doesn't like `<(…)`:

    dist/tools/radvd/radvd.sh: 19: Syntax error: "(" unexpected

Comment (miri64) [reply dist/tools/radvd/radvd.sh]: Ok.

Comment (miri64) [inline dist/tools/radvd/radvd.sh:19]: Some feedback on weather or not starting was successful would be nice.

File: dist/tools/radvd/radvd.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+PIDFILE="/tmp/radvd-${SUDO_USER}.pid"
+
+stop_radvd() {
+    if [ -f "${PIDFILE}" ]; then
+        PID=$(cat ${PIDFILE})
+    fi
+    if [ -n "${PID}" ]; then
+        kill ${PID}
+        rm ${PIDFILE}
+    fi
```
Comment (miri64) [inline dist/tools/radvd/radvd.sh:13]: Same here.

File: dist/tools/radvd/radvd.sh (hunk 1 above)
Comment (benpicco) [reply dist/tools/radvd/radvd.sh:20]: verbosity increased

**Comment by miri64** [comment]
Please squash

**Pull request merged**